diff --git a/node_modules/@ducdh-origin/react-native-twitter-signin/android/src/main/java/com/goldenowl/twittersignin/TwitterSigninModule.java b/node_modules/@ducdh-origin/react-native-twitter-signin/android/src/main/java/com/goldenowl/twittersignin/TwitterSigninModule.java
index 250cbc4..e710399 100644
--- a/node_modules/@ducdh-origin/react-native-twitter-signin/android/src/main/java/com/goldenowl/twittersignin/TwitterSigninModule.java
+++ b/node_modules/@ducdh-origin/react-native-twitter-signin/android/src/main/java/com/goldenowl/twittersignin/TwitterSigninModule.java
@@ -69,22 +69,24 @@ public class TwitterSigninModule extends ReactContextBaseJavaModule implements A
                 map.putString("name", session.getUserName());
                 map.putString("userID", Long.toString(session.getUserId()));
                 map.putString("userName", session.getUserName());
-                twitterAuthClient.requestEmail(session, new com.twitter.sdk.android.core.Callback<String>() {
-                    @Override
-                    public void success(Result<String> result) {
-                        map.putString("email", result.data);
-                        promise.resolve(map);
-                    }
-
-                    @Override
-                    public void failure(TwitterException exception) {
-                        map.putString("email", "COULD_NOT_FETCH");
-                        promise.reject(
-                                "COULD_NOT_FETCH",
-                                map.toString(),
-                                new Exception("Failed to obtain email", exception));
-                    }
-                });
+                promise.resolve(map);
+
+//                twitterAuthClient.requestEmail(session, new com.twitter.sdk.android.core.Callback<String>() {
+//                    @Override
+//                    public void success(Result<String> result) {
+//                        map.putString("email", result.data);
+//                        promise.resolve(map);
+//                    }
+//
+//                    @Override
+//                    public void failure(TwitterException exception) {
+//                        map.putString("email", "COULD_NOT_FETCH");
+//                        promise.reject(
+//                                "COULD_NOT_FETCH",
+//                                map.toString(),
+//                                new Exception("Failed to obtain email", exception));
+//                    }
+//                });
             }
 
             @Override